src/components/strategies/create/CreateOverlapping.tsx

```diff
@@ -42,7 +42,8 @@ const url = '/strategies/create/overlapping';
 export const CreateOverlapping: FC<Props> = (props) => {
   const { base, quote, order0, order1, marketPrice, spread } = props;
   const navigate = useNavigate({ from: url });
-  const { anchor, budget } = useSearch({ from: url });
+  const search = useSearch({ from: url });
+  const { anchor, budget } = search;
```

Comment:
why not also extract marketPrice here?

Reply:
There are multiple type of market price, the one from the search is the user market price and might not be there. We use different market price depending on the situation, and this calculation is done at parent component level.